web/app/js/components/Gateway.jsx

```diff
@@ -99,7 +116,15 @@ class Gateways extends React.Component {
           <Spinner />
         ) : (
           <div>
-            {noMetrics ? <div><Trans>noResourcesDetectedMsg</Trans></div> : null}
+            {noMetrics && multiclusterExists ? <div><Trans>noResourcesDetectedMsg</Trans></div> : null}
+            {(noMetrics && !multiclusterExists) &&
+            <Card>
+              <CardContent>
+                <Typography>To view gateway stats for your mesh, install the linkerd multicluster extension by running:</Typography>
+                <br />
+                <code>linkerd multicluster install | kubectl apply -f -</code>
+              </CardContent>
+            </Card>}
```

Comment:
Do you mind setting this up with translations? I believe most, if not all, text is currently using the `<Trans>` tags, so I think it'd be good practice to continue doing that whenever new text gets added, to avoid accumulating debt in that regard. `BUILD.md` has a new section explaining how to extract these strings to include them in the lingui json language assets.

Reply:
Good catch! I'll get that changed.